graphbolt/include/graphbolt/csc_sampling_graph.h

```diff
@@ -232,6 +238,34 @@ torch::Tensor Pick(
     int64_t offset, int64_t num_neighbors, int64_t fanout, bool replace,
     const torch::TensorOptions& options);
 
+/**
+ * @brief Picks a specified number of neighbors for a node per edge type,
+ * starting from the given offset and having the specified number of neighbors.
+ *
+ * @param offset The starting edge ID for the connected neighbors of the sampled
+ * node.
+ * @param num_neighbors The number of neighbors to pick.
+ * @param fanouts The number of edges to be sampled for each node per edge type.
+ * The value of it should be greater than or equal to 0 or -1. If -1 is given,
+ * it is equivalent to when the fanout is greater or equal to the number of
+ * eighbors and replacement is false, in which case all the neighbors will be
+ * selected. In contrast, a non-negative integer determines the minimum number
+ * of neighbors to select. The minimum value is determined by comparing the
+ * fanout value with the total number of neighbors available.
```

Comment:
```suggestion
   * The value of each fanout should be >= 0 or = 1.
   *   - When the value is -1, all neighbors will be chosen for sampling. It is equivalent to selecting all neighbors when the fanout is >= the number of neighbors (and replacement is set to false).
   *   - When the value is a non-negative integer, it serves as a minimum threshold for selecting neighbors.
```

Reply:
Let's update the @param fanout for Pick method above.

Reply:
Fixed